python/SignaliQ/client.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, config = None):
         self._current_dir = None
         self._params = None
 
-        self._config = config if config else self._build_config_from_file()
+        config_file = path.join(self._get_current_dir(), "config.ini")
```

Comment:
Can you change the name of _get_current_dir to _get_file_dirname (current dir is confusing, as that concept refers to the shell's current working directory, which is not necessarily the same as the directory path of this file)?

Reply:
Changed the name to _get_file_dirname